src/lib/es2015.iterable.d.ts

```diff
@@ -231,243 +231,14 @@ interface Int8Array {
     values(): IterableIterator<number>;
 }
 
-interface Int8ArrayConstructor {
-    new (elements: Iterable<number>): Int8Array;
+interface TypedArrayConstructor<T> {

```

Comment:
I would rather we leave out `TypedArrayConstructor` entirely and leave the individual constructor interfaces.